config.toml

```diff
@@ -7,7 +7,7 @@ taxonomies = [
     {name = "tags", feed = true},
 ]
 [extra]
-even_title = "Cufezhusy's Blog"
+even_title = "yihong0618's Blog"
 even_menu = [
     {url = "$BASE_URL", name = "Home"},
     {url = "$BASE_URL/tags/yi-xie-ji-lu/", name = "一些记录"}},
```

Comment:
also here

Reply:
Good point, yes let me try fix it now :).